index.d.ts

```diff
@@ -83,6 +83,11 @@ declare class QuickLRU<KeyType, ValueType>
 	*/
 	clear(): void;
 
+	/**
+	Resize the cache to fit the given maximum number of items.
```

Comment:
index.d.ts and the readme should be completely in sync text-wise, meaning this and the others should use the same text.

Reply:
It looks like you're in the process of updating the PR - I'll leave you to it. lmk if you'd like me to make such changes.

Reply:
I'm done. Feel free to make the change.

Reply:
Done!